sites/funmooc/src/backend/locale/fr/LC_MESSAGES/django.po

```diff
@@ -21,51 +21,71 @@ msgstr ""
 msgid "Unsorted uploads are not allowed."
 msgstr "Les téléchargements non classés sont interdits"
 
-#: funmooc/settings.py:219
+#: funmooc/settings.py:222
+msgid "Dashboard"
+msgstr ""
+
+#: funmooc/settings.py:223
+msgid "{base_url:s}/dashboard"
+msgstr ""
+
+#: funmooc/settings.py:226
 msgid "Profile"
 msgstr "Profil"
 
-#: funmooc/settings.py:220
+#: funmooc/settings.py:227
+msgid "{base_url:s}/u/(username)"
```

Comment:
this pattern looks weird (username)?

Reply:
This is something use by the frontend to be able to dynamically replace `(username)` string by the authenticated username.